src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "      ____        _        \n"
+                + "     |  _ \\ _   _| | _____ \n"
+                + "     | | | | | | | |/ / _ \\\n"
+                + "     | |_| | |_| |   <  __/\n"
+                + "     |____/ \\__,_|_|\\_\\___|\n";
+
+        String hello = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" + logo +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+
+        System.out.println(hello);
+
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> task_list = new ArrayList<Task>();
+
+        try {
+            task_list = loadFile();
+        }
+
+        catch (IOException ioException) {
+            System.out.println(ioException.getMessage());
+        }
+
+        Boolean bye = false;
+
+        while (sc.hasNextLine() && !bye) {
+            String command = sc.nextLine();
+                try {
+                    Duke.TaskIdentifier(command, task_list);
+                    createFile(task_list);
+                }
+
+                catch (DukeInvalidCommandException e) {
+                    System.out.println(e.toString());
+                }
+
+                catch (DukeIncompleteCommandException e) {
+                    System.out.println(e.toString());
+                }
+
+                catch (IOException ioException) {
+                    System.out.println(ioException.getMessage());
+                }
+            }
+        }
+
+    public static void TaskIdentifier(String command, ArrayList<Task> task_list) throws DukeInvalidCommandException, DukeIncompleteCommandException{
+            String task_type = command.split(" ")[0];
+            String task_details = "";
+            String task_info = "";
+
+            if (!task_type.equals("todo") && !task_type.equals("event") && !task_type.equals("deadline")
+                    && !task_type.equals("list") && !task_type.equals("bye") && !task_type.equals("done")
+                    && !task_type.equals("delete")) {
+                throw new DukeInvalidCommandException("    ____________________________________________________________\n" +
+                        "     ☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" +
+                        "    ____________________________________________________________");
```

Comment:
I think can try to save the line string as a constant named as something like "LINE", then instead of copying and pasting repeatedly, can call the constant when you want to draw a line